lib/uring/uring.mli

```diff
@@ -145,6 +145,18 @@ val openat2 : 'a t ->
     @param perm sets the access control bits for newly created files (subject to the process's umask)
     @param resolve controls how the pathname is resolved. *)
 
+val linkat : 'a t -> ?target_dir_fd:Unix.file_descr -> ?link_dir_fd:Unix.file_descr -> follow:bool -> target:string -> link_name:string -> 'a -> 'a job option
+(** [link t ?follow ~target ~link_name] creates a new hard link. If the path of the link
```

Comment:
```suggestion
(** [linkat t ~follow ~target ~link_name] creates a new hard link. If the path of the link
```
I'm not keen on the `target` and `link` names. They would make sense for symlinks, but either could be described as the "target" or as the "link" here. The Linux man-page uses `old` and `new` which seems clearer to me.